tests/kubernetes/README.md

```diff
@@ -13,30 +13,48 @@ To build this image locally and optionally push to the SkyPilot registry, run:
+./build_image.sh -p -g
 ```
 
 ## Running a local development cluster
 We use (kind)[https://kind.sigs.k8s.io/] to run a local Kubernetes cluster 
-for development.
+for development. To create a local development cluster, run:
 
 ```bash 
 sky local up
 ```
 
 ## Running a GKE cluster
-1. Make sure ports 30000-32767 are open in your node pool VPC's firewall.
-2. Create a GKE cluster with at least 1 node. We recommend creating nodes with at least 4 vCPUs.
+1. Create a GKE cluster with at least 1 node. We recommend creating nodes with at least 4 vCPUs.
    * Note - only GKE standard clusters are supported. GKE autopilot clusters are not supported.
+   * Tip - to create an example GPU cluster for testing, this command will create a 2 node cluster with 1x T4 and another with 1x V100:
+     ```bash
+     gcloud beta container --project "skypilot-375900" clusters create "testcluster" --zone "us-central1-c" --no-enable-basic-auth --cluster-version "1.27.2-gke.1200" --release-channel "regular" --machine-type "n1-standard-8" --accelerator "type=nvidia-tesla-t4,count=1" --image-type "COS_CONTAINERD" --disk-type "pd-balanced" --disk-size "100" --metadata disable-legacy-endpoints=true --scopes "https://www.googleapis.com/auth/devstorage.read_only","https://www.googleapis.com/auth/logging.write","https://www.googleapis.com/auth/monitoring","https://www.googleapis.com/auth/servicecontrol","https://www.googleapis.com/auth/service.management.readonly","https://www.googleapis.com/auth/trace.append" --num-nodes "1" --logging=SYSTEM,WORKLOAD --monitoring=SYSTEM --enable-ip-alias --network "projects/skypilot-375900/global/networks/default" --subnetwork "projects/skypilot-375900/regions/us-central1/subnetworks/default" --no-enable-intra-node-visibility --default-max-pods-per-node "110" --security-posture=standard --workload-vulnerability-scanning=disabled --no-enable-master-authorized-networks --addons HorizontalPodAutoscaling,HttpLoadBalancing,GcePersistentDiskCsiDriver --enable-autoupgrade --enable-autorepair --max-surge-upgrade 1 --max-unavailable-upgrade 0 --enable-managed-prometheus --enable-shielded-nodes --node-locations "us-central1-c" && gcloud beta container --project "skypilot-375900" node-pools create "v100" --cluster "testcluster" --zone "us-central1-c" --machine-type "n1-standard-8" --accelerator "type=nvidia-tesla-v100,count=1" --image-type "COS_CONTAINERD" --disk-type "pd-balanced" --disk-size "100" --metadata disable-legacy-endpoints=true --scopes "https://www.googleapis.com/auth/devstorage.read_only","https://www.googleapis.com/auth/logging.write","https://www.googleapis.com/auth/monitoring","https://www.googleapis.com/auth/servicecontrol","https://www.googleapis.com/auth/service.management.readonly","https://www.googleapis.com/auth/trace.append" --num-nodes "1" --enable-autoupgrade --enable-autorepair --max-surge-upgrade 1 --max-unavailable-upgrade 0 --node-locations "us-central1-c"
```

Comment:
Not sure if we want to show the project_id of our GCP project. Is it possible to remove it from the git histroy?

Reply:
Also, it would be easier to make the project-id and cluster name configurable with env vars. Using `PROJECT_ID` and `CLUSTER_NAME`

Reply:
Trying out this command with the following version but get the following error occurs:
```
Google Cloud SDK 443.0.0
beta 2023.08.11
core 2023.08.11
```

```
ERROR: (gcloud.beta.container.clusters.create) could not parse resource []
```

Reply:
Great points!  Thanks, I'll clean up and remove our project_id.

The command seems to work for me on:
```
Google Cloud SDK 432.0.0
beta 2023.05.19
core 2023.05.19
gke-gcloud-auth-plugin 0.5.3
```
Let me try to upgrade my gcloud and try again.

Reply:
Hmm, even after upgrading to the latest gcloud, the command seems to work for me.
```
Google Cloud SDK 443.0.0
beta 2023.08.11
core 2023.08.11
gke-gcloud-auth-plugin 0.5.5
```
Here are my [logs](https://gist.github.com/romilbhardwaj/9dcc71d871c044599f176dc24ce729aa). Are you sure you're pasting the command correctly?

Reply:
I've updated this command to use envvars to detect project id and set cluster name :)